k8s/examples/kafka/01-rbac.yaml

```diff
@@ -0,0 +1,38 @@
+---
+# Source: kafka/templates/rbac.yaml
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kafka
+  namespace: default
+
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: kafka
+  namespace: default
```

Comment:
Could we please remove the namespace: default.

Reply:
removed